### DISCUSSION TITLE
Failed to identify any smart meter on your account

### DISCUSSION THREAD

**Discussion by xtalker** [mjmeli/ha-duke-energy-gateway#135] [Q&A]
Anyone else seeing this in the HA log?:

`pyduke_energy.errors.DukeEnergyError: Failed to identify any smart meter with gateway access on your account`

This integration has been working for months but started throwing this error on 9/21.  I did try the delete/reinstall integration trick, numerous gateway and HA restarts etc. but no luck.

Full log entry:
`This error originated from a custom integration.

Logger: homeassistant.config_entries
Source: custom_components/duke_energy_gateway/__init__.py:54
Integration: Duke Energy Gateway (documentation, issues)
First occurred: September 26, 2022 at 5:25:45 PM (4 occurrences)
Last logged: 11:21:23 AM

Error setting up entry bob@steinbeiser.com for duke_energy_gateway
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 353, in async_setup
    result = await component.async_setup_entry(hass, self)
  File "/config/custom_components/duke_energy_gateway/__init__.py", line 54, in async_setup_entry
    selected_meter, selected_gateway = await client.select_default_meter()
  File "/usr/local/lib/python3.10/site-packages/pyduke_energy/client.py", line 175, in select_default_meter
    raise DukeEnergyError(
pyduke_energy.errors.DukeEnergyError: Failed to identify any smart meter with gateway access on your account
`

**Comment by mjmeli** [comment]
Are you running the latest version of the integration (v0.1.2)? Duke made a change to their API recently that caused this error and that new version has a fix.
